internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestHybrid.java

```diff
@@ -1467,6 +1467,7 @@ public void testVersionSwapDeferredWithHybrid() throws Exception {
               .setHybridOffsetLagThreshold(streamingMessageLag)
               .setPartitionCount(1));
       Assert.assertFalse(response.isError());
+      // venice.getPubSubClientProperties();
```

Comment:
we should copy pubsub properties into VPJ props

Reply:
yea adding them in defaultVPJProps()